projects.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+    <meta charset="utf-8">
+    <title>PROJECTS</title>
+    <link rel=StyleSheet href="styles/stylesheet.css">
+  </head>
+
+  <body>
+    <header id="home-header">
+      <section class="name-role">
+      <h1 >Rana Sulaiman</h1>
+      <h2 >Full Stack Developer</h2>
+    </section>
+
+      <nav class="main-nav" role="navigation">
+        <ul >
+          <li><a href="home.html" title="link to home  page">HOME</a></li>
+          <li><a href="aboutme.html" title="link to portfolio page">ABOUT ME</a></li>
+          <li><a href="journal.html" title="link to journal page">JOURNAL</a></li>
+          <li><a href="projects.html" title="link to projects page">PROJECTS</a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+
+
+      <article class="projects">
+        <h1>LATEST PROJECTS</h1>
+
+        <section>
+          <a href="https://github.com/RanaSulaiman/BankAccounts">Bank Accounts</a>
+          <!-- <h2>project #1</h2> -->
+          <p>
+            1- Health goth migas celiac, trust fund ramps before they sold out +1 fanny pack microdosing cornhole polaroid hashtag. Af artisan pork belly, messenger bag prism tbh 90's health goth. Heirloom kinfolk tilde try-hard, man bun bespoke typewriter food truck forage hella hexagon
+           </p>
+        </section>
+        <section>
+          <a href="https://github.com/RanaSulaiman/ride-share-two">Ride Share</a>
+          <p>
+            Health goth migas celiac, trust fund ramps before they sold out +1 fanny pack microdosing cornhole polaroid hashtag. Af artisan pork belly, messenger bag prism tbh 90's health goth. Heirloom kinfolk tilde try-hard, man bun bespoke typewriter food truck forage hella hexagon S
```

Comment:
I see you starting to number your projects (and using CSS to insert the content ```project #``` in front of these, which is kind of awesome), but why did you leave off the numbers for 2-4?